### PR TITLE
add italian translation

### PR DESCRIPTION
**Pull request by fpisu-sardit**
Add Italian translation.

### CODE REVIEW

File: backend/i18n/source/backend_it.json [hunk 1]
```diff
@@ -23,24 +23,6 @@
   "apps.roles.nameAlreadyExists": "Esiste già un ruolo con lo stesso nome.",
   "apps.roles.usedRoleByClientsNotRemovable": "Non è possibile rimuovere un ruolo quando è assegnato ad un ruolo.",
   "apps.roles.usedRoleByContributorsNotRemovable": "Non è possibile rimuovere un ruolo quando questo è assegnato ad un collaboratore.",
-  "aspnet_annotations_AbsoluteUrl": "Il campo {0} deve essere un URL assoluto.",
```
Comment (SebastianStehle) [inline backend/i18n/source/backend_it.json]: Sorry, I only meant that you should remove the aspnet_annotations_XXX translations, NOT `aspnet_identity_XXX`

**Review by SebastianStehle — CHANGES_REQUESTED** [review]
Have you executed translate.bat? Because the resx files are not updated.

File: backend/i18n/source/backend_it.json (hunk 1 above)
Comment (fpisu-sardit) [reply backend/i18n/source/backend_it.json]: I have removed all labels that weren't in the origin file

Comment (fpisu-sardit) [reply backend/i18n/source/backend_it.json]: Please add the missing label

Comment (SebastianStehle) [reply backend/i18n/source/backend_it.json]: You just have to get the old translations back and rename the keys, see: https://docs.squidex.io/01-getting-started/contributing-and-developing/translating#how-to-fix-errors

**Pull request merged**